libazure/2D.h

```diff
@@ -448,7 +448,8 @@ class PathSink : public RefCounted<PathSink>
    */
   virtual void Close() = 0;
   /** Add an arc to the current figure */
-  virtual void Arc(const Point &aOrigin, float aRadius, float aStartAngle,
+  virtual void Arc(const Point &aOrigin, float aRadiusX, float aRadiusY,
+                   float aRotationAngle, float aStartAngle,
```

Comment:
This looks like it should be upstreamed? i'm actually surprised moz2d didn't have a way to do this...